.eslintignore

```diff
@@ -6,6 +6,7 @@ docs/public
 storybook-static
 built-storybooks
 lib/cli/test
+examples/ember-cli
```

Comment:
why ?

Reply:
```
Unable to resolve path to module 'htmlbars-inline-precompile'  import/no-unresolved
```

We use a babel module to precompile these and the linter looks for resolved dependencies, do we have a way around this?

Reply:
Yes: https://github.com/storybooks/storybook/blob/master/MIGRATION.md#from-version-34x-to-40x

Please, add an `examples/ember-cli/.eslintrc.js` file with `settings: {'import/core-modules' [<list of injected modules>]},`

Reply:
okay I did this thank you! :)